flow/layers/picture_layer.cc

```diff
@@ -49,15 +49,10 @@ void PictureLayer::Paint(PaintContext& context) const {
       context.leaf_nodes_canvas->getTotalMatrix()));
 #endif
 
-  if (context.raster_cache) {
-    const SkMatrix& ctm = context.leaf_nodes_canvas->getTotalMatrix();
-    RasterCacheResult result = context.raster_cache->Get(*picture(), ctm);
-    if (result.is_valid()) {
-      TRACE_EVENT_INSTANT0("flutter", "raster cache hit");
-
-      result.draw(*context.leaf_nodes_canvas);
-      return;
-    }
+  if (context.raster_cache &&
+      context.raster_cache->Draw(*picture(), *context.leaf_nodes_canvas)) {
+    TRACE_EVENT_INSTANT0("flutter", "raster cache hit");
```

Comment:
This trace event no longer makes sense as it doesn't surround any significant piece of code. Perhaps do this tracing inside the raster cache itself?